node/src/store/redb_store.rs

```diff
@@ -35,6 +35,8 @@ const SCHEMA_VERSION_TABLE: TableDefinition<'static, (), u64> =
     TableDefinition::new("STORE.SCHEMA_VERSION");
 const RANGES_TABLE: TableDefinition<'static, &str, Vec<(u64, u64)>> =
     TableDefinition::new("STORE.RANGES");
+const LIBP2P_IDENTITY_TABLE: TableDefinition<'static, &str, &[u8]> =
```

Comment:
I prefer something like `TableDefinition<'static, (), &[u8]>` and store the private part only